src/autoyast-rnc/kdump.rnc

```diff
@@ -2,18 +2,22 @@ default namespace = "http://www.suse.com/1.0/yast2ns"
 namespace a = "http://relaxng.org/ns/compatibility/annotations/1.0"
 namespace config = "http://www.suse.com/1.0/configns"
 
+include "common.rnc"
 
 kdump =
   element kdump {
+    MAP,
```

Comment:
Is the indentation OK? I would say that it is slightly different to the used for the `general` section.

Reply:
I worry this use old ycp indentation rule with 4 width and every second indent is tab, so it creates some fancy indentation. If you are fine, I can unify it to have 2 space everywhere

Reply:
Yes, I am fine with that. Thanks.